bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java

```diff
@@ -49,6 +54,12 @@
 
     private static HttpClient client = new HttpClient(new SslContextFactory());
 
+    private static String proxyHost = null;
+    private static int proxyPort = 80;
+    private static String proxyUser = null;
+    private static String proxyPassword = null;
+    private static String nonProxyHosts = null;
```

Comment:
I noticed that too. In this case the global statics will only be filled by also globally defined system properties. Otherwise I would prefer HttpUtils not to be static which is API breaking...